master/internal/configpolicy/task_config_policy.go

```diff
@@ -228,6 +228,8 @@ func MergeWithInvariantExperimentConfigs(ctx context.Context, workspaceID int,
 	if !reflect.DeepEqual(originalConfig, config) {
 		log.Warnf("some fields were overridden by admin %s config policies", scope)
 	}
+
+	config = schemas.WithDefaults(config)
```

Comment:
To ensure I understand ... we call `WithDefaults` here because ... ?
- we do not want to call `WithDefaults` on our invariant config
- we still need the final config to be valid